brave/brave-core
# Add declared geo to brave://rewards-internals

Resolves https://github.com/brave/brave-browser/issues/25656

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Enable Rewards and declare geo
- Visit brave://rewards-internals
- Verify that declared geo matches what's shown on the internals page
